velox/expression/tests/CastExprTest.cpp

```diff
@@ -29,13 +31,25 @@
 using namespace facebook::velox;
 namespace facebook::velox::test {
 namespace {
+template <typename TExecParams>
+struct FailFunction {
```

Comment:
Would you add fail function in a separate PR then rebase this one?

Reply:
The consensus in the Prestissimo WG meeting last week was to add the `fail()` function in Prestissimo instead of Velox.
Links to the notes here https://docs.google.com/document/d/1oL0KVf_02_P_kIfNs2oZVwB60_PqHBNou6MzRAS1fbM/edit#heading=h.o85nkj3afpgg
There is an issue open here https://github.com/prestodb/presto/issues/20742 if you would like to share your thoughts.